linode/manager
# chore: Clean up how dev tools are initialized

## Description 📝

- We used callbacks to initialize our devtools and I don't think we need to
 
## Major Changes 🔄
- Use more simple logic to conditionally enable dev tools without the use of callbacks
  - This should make things cleaner and easier follow

## How to test 🧪
- Verify dev tools still work as expected in local development (`yarn dev`)
- Verify dev tools are not shown in production builds (`yarn workspace linode-manager run build && yarn workspace linode-manager run start:ci`)
